### PR TITLE
Add some documentation for paradigms inherited from Vulcan

### PR DESCRIPTION
**Pull request by jpaddison3**
And some general cleaning up.

Already reviewed in: https://github.com/centre-for-effective-altruism/EAForum/pull/183

### CODE REVIEW

**Comment by jpaddison3** [comment]
Requesting a review from @Discordius I guess? It's already been reviewed, move it's good for this review to be a "FYI" for someone at lesswrong

**Review by Discordius — APPROVED** [review]

**Comment by jpaddison3** [comment]
@Discordius I can't merge this because of the circleci builds still being there. I thought we fixed those, and why didn't the real tests run? Is it related to the fact that this is submitted against master?
Anyway, I also think once I've already merged something into ea-master, you can assume I want to merge this one.

**Comment by Discordius** [comment]
I've clicked around for like 10 minutes trying to figure out how to turn off the CircleCI check, but can't find it. I've now deactivated the requirement to have checks pass to merge to master.

**Pull request merged**
